content/applications/services/helpdesk.rst

```diff
@@ -143,6 +140,43 @@ drop-down menu. Doing so opens a new page where the selected tickets are listed
 :guilabel:`Similarity` rating. From here, click either :ref:`Merge <data_cleaning/merge-records>` to
 combine the tickets, or :guilabel:`DISCARD`.
 
+Convert tickets to opportunities
+================================
+
+Some tickets may be better handled by the sales team, rather than the support team. In this case,
+tickets can be converted to *opportunities* and assigned to a sales team for follow-up.
+
+.. important::
+   This feature is **only** available if the **CRM** app is installed.
+
+To convert a ticket to an opportunity, first navigate to a ticket, either from a team's pipeline,
+or by navigating to :menuselection:`Helpdesk app --> Tickets` and clicking a ticket to open it.
+
+At the top of the ticket, click the :guilabel:`Convert to Opportunity` button.
+
+.. note::
+   If :doc:`leads <../sales/crm/acquire_leads/convert>` are enabled on the **CRM** app, tickets are
+   converted to *leads*, and the button reads :guilabel:`Convert to Lead`.
+
+This opens the :guilabel:`Convert to Opportunity` pop-up. Fill in or select the following
+information on the pop-up:
+
+- Customer: select whether to :guilabel:`Create a new customer`, :guilabel:`Link to an existing
+  customer`, or :guilabel:`Do not link to a customer`. If :guilabel:`Link to a customer` is chosen,
+  select the appropriate customer name from the :guilabel:`Customer` drop-down.
+
+- Sales Team: select a :guilabel:`Sales Team` and :guilabel:`Salesperson` to assign the opportunity
+  to.
+
+.. image:: helpdesk/convert-to-opp.png
+   :alt: The convert to opportunity pop-up window.
+
+After completing the form, click :guilabel:`Convert to Opportunity`. Doing so creates a new
+opportunity. The original ticket is linked in the chatter of the new opportunity for traceability.
```

Comment:
```suggestion
opportunity in the **Sales** app. The original ticket is linked in the chatter of the new opportunity for traceability.
```

Reply:
Opps and leads are in the CRM app, but I will add this :+1: